.travis.yml

```diff
@@ -6,5 +6,5 @@ rvm:
   - '2.2.4'
   - '2.3.0'
 script:
-  - bundle exec rubocopter --commit origin/master
+  - gem install rubocop
```

Comment:
I think there is a specific step for installing things. Once installed you should run `rubocop` in this step.